.circleci/config.yml

```diff
@@ -229,6 +229,13 @@ jobs:
       - store_artifacts:
           path: test-artifacts
           destination: test-artifacts
+      # important: generate sesify viz AFTER uploading builds as artifacts
```

Comment:
I'm guessing this needs to be run after because otherwise it'd override the "real" build artifacts. I guess this is the main obstacle to running this in parallel - writing the output to a different location, so it can be done earlier.

Reply:
Correct. Could add a flag to build dist to a different dir. Though worth noting that all jobs past `all-tests-pass` are optional and don't block merges